src/js/components/listPagesUtils/Tabs.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import useQueryParams from 'hooks/useQueryParams';
+import Translate from 'utils/Translate';
+
+const Tabs = ({ config }) => {
```

Comment:
Is the name of it too "generic"? Could we get a better name for it? Or do you feel like it is ok?

Reply:
Initially it was named as "ProductSupplierTabs", but I got a suggestion from @drodzewicz, that we could eventually use this component for further redesigns (e.g. PO), hence the name should be more generic.